lib/sqlalchemy/sql/sqltypes.py

```diff
@@ -1798,11 +1802,20 @@ def __init__(
           to compare values of this type.  If left as ``None``,
           the Python "equals" operator is used to compare values.
 
+        :param impl: Any implementation class that support storing
+          binary object. (e.g. :class: `.mysql.LONGBLOB`).
+          If let as ``None``, the impl would use class's default impl
+          `:class:`.LargeBinary`
+
         """
         self.protocol = protocol
         self.pickler = pickler or pickle
         self.comparator = comparator
-        super(PickleType, self).__init__()
+
+        if impl:
+            self.impl = to_instance(impl)
+        else:
+            super(PickleType, self).__init__()
```

Comment:
I think it's best to always call super init, even in this case it would cause needles instantiation

Reply:
Fixed.